quality-assurance/test-strategy.md

```diff
@@ -70,6 +70,13 @@ This section outlines the test types that need to be considered when preparing a
   * Pester tests to validate the environment is as expected.
   * Smoke and/or regression tests to ensure the product(s) are working in the environment as expected.
 
+### Disaster Recovery
+
+The disaster recovery plan should be tested for a new or significantly changed system.  This will normally consist of a drill, whereby the disaster scenario(s) are simulated, and the written disaster recovery plan is followed.  This is to verify:
+
+* the plan is complete and accurate, and following it leads to a fully recovered and functional system.
```

Comment:
the adherence to the plan will result in the Application / system being placed into its desired state, with in the time stated for recovery to that state.